.env

```diff
@@ -0,0 +1,6 @@
+# Domain name for service. Should be equal with name in generated ssl-certificate
+NGINX_DEBUG=True
+# Domain name for service. Should be equal with name in generated ssl-certificate
+NGINX_VHOSTNAME=demo.kqueen.net
+# Directory path for certificates in container.Finally it look like $NGINX_SSL_CERTIFICATE_DIR/$NGINX_VHOSTNAME
+NGINX_SSL_CERTIFICATE_DIR=/mnt/letsencrypt
```

Comment:
Should name be more specific? what if some other images will need env variables?

Reply:
.env file its common default file for running compose-deloyments